### PR TITLE
Standardize mobile crash report setup

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do? What is the motivation?

As per discussion with @MaelNamNam, this standardizes the mobile crash reporting setup pages across all frameworks:
- All pages now have these sections:
  - Overview
  - Setup
  - Get deobfuscated stack traces
  - Limitations
  - Test your implementation

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Comment by rtrieu** [comment]
Hi @MaelNamNam , this PR is ready for review!

File: content/en/real_user_monitoring/error_tracking/mobile/expo.md [hunk 1]
```diff
@@ -80,15 +68,33 @@ Run `eas secret:create` to set `DATADOG_SITE` to the host of your Datadog site,
...
 Source maps, mapping files, and dSYM files are limited to **500** MB each.
 {{< /site-region >}}
 {{< site-region region="ap1" >}}
-Source maps, mapping files, and dSYM files  are limited to **500** MB each.
+Source maps, mapping files, and dSYM files are limited to **500** MB each.
 {{< /site-region >}}
 
+## Test your implementation
+
+## Additional configuration options
```
Comment (estherk15) [inline content/en/real_user_monitoring/error_tracking/mobile/expo.md:96]: Back to back H2 headers, is this intentional?

File: content/en/real_user_monitoring/error_tracking/mobile/reactnative.md [hunk 2]
```diff
@@ -142,21 +158,23 @@ const crashApp = () => {
...
+First, you need to install `@datadog/datadog-ci` as a dev dependency to your project:
 
 ```bash
 yarn add -D @datadog/datadog-ci
 # or
 npm install --save-dev @datadog/datadog-ci
 ```
 
-#### Automatically on each release build (React Native >= 0.69)
+{{% collapse-content title="Automatically on each release build (React Native >= 0.69)" level="h5" %}}
```
Comment (estherk15) [inline content/en/real_user_monitoring/error_tracking/mobile/reactnative.md:177]: Oooooh, collapsible config! It's so cool!!!

File: content/en/real_user_monitoring/error_tracking/mobile/roku.md [hunk 3]
```diff
@@ -40,7 +40,28 @@ If you have not set up the Roku SDK yet, follow the [in-app setup instructions][
 
 For any given error, you can access the file path, line number, and a code snippet for each frame of the related stack trace.
 
-## Forward errors to Datadog
+## Limitations
+
+**Note**: Crash reporting on Roku doesn't yet support stacktraces.
```
Comment (estherk15) [inline content/en/real_user_monitoring/error_tracking/mobile/roku.md:45]: Non-blocking

```suggestion
Crash reporting on Roku doesn't yet support stacktraces. 
```

**Review by estherk15 — APPROVED** [review]

File: content/en/real_user_monitoring/error_tracking/mobile/expo.md (hunk 1 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/error_tracking/mobile/expo.md:96]: Yes it is intentional but the Test your implementation section shouldn't be empty. Thanks for calling this out, I'll need to get it filled in!

File: content/en/real_user_monitoring/error_tracking/mobile/reactnative.md (hunk 2 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/error_tracking/mobile/reactnative.md:177]: Ya it's @brett0000FF's creation. It is very cool and improves readability for pages like this!

**Review by rtrieu — COMMENTED** [review]
Commenting

**Pull request merged**